cli/client/main.go

```diff
@@ -556,16 +543,16 @@ func mainWithStatusCode() int {
 		tlsConf,
 		&qconf)
 	if err != nil {
-		if transportErr, ok :=  err.(*quic.TransportError); ok {
+		if transportErr, ok := err.(*quic.TransportError); ok {
 			if transportErr.ErrorCode.IsCryptoError() {
 				if tty == nil {
 					log.Error().Msgf("insecure server cert in non-terminal session, aborting")
 					return -1
 				}
 				if _, ok = knownHosts[hostname]; ok {
-					log.Error().Msgf("The server certificate cannot be verified using the one installed in %s. " + 
```

Comment:
I am okay to reformat the code in a standard manner but could we do that in another PR ? :-)

Reply:
Ouch! That was not intended — you're absolutely right about keeping one topic per PR.

Reply:
Done! I don't want to monopolize your time, so I won't re-re-request a review at this time.

Reply:
You can request it anytime, don't worry.